requirements/requirements.txt

```diff
@@ -18,6 +18,7 @@ django-kombu==0.9.1
 django-rosetta==0.7.1
 django-tastypie==0.12.1
 djtables==0.1.1
+elasticsearch==0.4.4
```

Comment:
fyi @dimagi/team-commcare-hq I'm looking into how hard it's going to be to rewrite all our ES code using this module. this is the officially supported one by elastic and is much more active than `rawes`. I also suspect that using it will make upgrading to `1.x` significantly easier.